fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -532,6 +534,8 @@ public class SessionVariable implements Serializable, Writable {
     @VariableMgr.VarAttr(name = EXTRACT_WIDE_RANGE_EXPR, needForward = true)
     public boolean extractWideRangeExpr = true;
 
+    @VariableMgr.VarAttr(name = ENABLE_VECTORIZED_ENGINE)
```

Comment:
```suggestion
    @VariableMgr.VarAttr(name = ENABLE_VECTORIZED_ENGINE, expType = ExperimentalType.EXPERIMENTAL_ONLINE)
```

Reply:
Nothing changed?

Reply:
@morningman Why it's EXPERIMENTAL_ONLINE? Should we add another expType `DEPRACATED` and mark this variable?